substrate/docs/deprecation-process.md

```diff
@@ -0,0 +1,62 @@
+# Deprecation Process
+
+This deprecation process makes sense while we don’t use [SemVer](https://semver.org/).
+After that, this process will most likely change.
+As deprecation and removal of existing features can happen on any release, we need to be mindful that external builders could be impacted by the changes we make.
+The deprecation process tries to mitigate this impact, while still keeping the developer experience, the DevEx, as smooth as possible.
+
+First of all we, need to create a new issue with the label `I11-deprecation` and add it to the [Runtime / FRAME](https://github.com/orgs/paritytech/projects/40) project. This will make sure that the issue is added to the [Deprecation board](https://github.com/orgs/paritytech/projects/40/views/12) for correct tracking.
```

Comment:
What if other components are being deprecated? (e.g https://github.com/paritytech/polkadot-sdk/blob/4a235469fd7ec4c6575862b0f6428f3601a59b3e/substrate/client/chain-spec/src/chain_spec.rs#L456)

Or maybe is this guide only for runtime/frame?

Reply:
Oops, bad copy paste. This process has initially been developed for FRAME, but the idea is to reuse it to the entire Substrate.